src/graphql/queries/colony.graphql

```diff
@@ -85,3 +85,31 @@ query GetColonyMemberInvite($id: ID!) {
     invitesRemaining
   }
 }
+
+query GetColonyTokensWithLockedStates($name: String!) {
+  getColonyByName(name: $name) {
```

Comment:
we really do hide this data under 3 layers of nested `items`, good job getting this data.
I can only think of getting one level of nesting out is if you call `getColony` with its address instead of getting it by name, this will rid us of the first `items`